k8s/namespaces/sscs/sscs-tribunals-api/ithc.yaml

```diff
@@ -4,28 +4,14 @@ metadata:
   name: sscs-tribunals-api
   namespace: sscs
 spec:
-  releaseName: sscs-tribunals-api
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/sscs-tribunals-api
   values:
     java:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/sscs/tribunals-api:prod-c57e19c-20220506140005 #{"$imagepolicy": "flux-system:sscs-tribunals-api"}
```

Comment:
Same with this, are you putting it now to overwrite at some point?

Reply:
Yes. Leaving it to flux to maintain the version on the env image tag.